aviatrix/resource_aviatrix_edge_platform.go

```diff
@@ -630,6 +638,12 @@ func resourceAviatrixEdgePlatformCreate(ctx context.Context, d *schema.ResourceD
 		}
 	}
 
+	// set the advertised spoke cidr routes
+	err := editAdvertisedSpokeRoutesWithRetry(client, gatewayForGatewayFunctions, d)
+	if err != nil {
+		return diag.Errorf("failed to edit advertised spoke vpc routes of spoke gateway: %s due to: %s", gatewayForGatewayFunctions.GwName, err)
```

Comment:
```suggestion
		return diag.Errorf("failed to edit advertised spoke vpc routes of spoke gateway %q: %s", gatewayForGatewayFunctions.GwName, err)
```

Quoting names in error strings tends to make them easier to read and removes possible ambiguity.